### PR TITLE
Fix Stenzek.DuckStation version

### PR DESCRIPTION
**Pull request by rakleed**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/129689)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with:
>

**Comment by stephengillie** [comment]
Manual Validation ended with:
>

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 1:06:05 PM:  > 2023-12-08 13:06:04.754 [CLI ] Package hash verification failed. SHA256 in manifest [7c3da21008092c067ada06c1a9362ab73f9fe541857c855647a5406af59a1789] does not match download [75e241998f965031f359c2920bef6257dabf22fd501a8d69c5c20631f7fc843a]

**Comment by stephengillie** [comment]
Pending Manual Validation.

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 10:44:57 AM:  2023-12-12 10:44:56.432 [CLI ] Package hash verification failed. SHA256 in manifest [75e241998f965031f359c2920bef6257dabf22fd501a8d69c5c20631f7fc843a] does not match download [d6c260def3cb6e9684ae8702f73edb91b19c8aa72efa3acdc2e642b4b3162d2c]

File: manifests/s/Stenzek/DuckStation/0.1.6163/Stenzek.DuckStation.installer.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 - InstallerUrl: https://github.com/stenzek/duckstation/releases/download/latest/duckstation-windows-x64-release.zip
   Architecture: x64
   InstallerType: zip
   NestedInstallerType: portable
   NestedInstallerFiles:
   - RelativeFilePath: duckstation-qt-x64-ReleaseLTCG.exe
-    PortableCommandAlias: duckstation
-  InstallerSha256: BE682525515184C7CD829836014EA06569EA215930C4D796A25FD0B33CF06DE9
+    PortableCommandAlias: DuckStation
+  InstallerSha256: 75E241998F965031F359C2920BEF6257DABF22FD501A8D69C5C20631F7FC843A
```
Comment (stephengillie) [inline manifests/s/Stenzek/DuckStation/0.1.6163/Stenzek.DuckStation.installer.yaml:14]:
```suggestion
  InstallerSha256: D6C260DEF3CB6E9684AE8702F73EDB91B19C8AA72EFA3ACDC2E642B4B3162D2C
```

**Comment by stephengillie** [comment]
> 2:27:39 PM: 2023-12-21 14:27:37.963 [CLI ] Package hash verification failed. SHA256 in manifest [d6c260def3cb6e9684ae8702f73edb91b19c8aa72efa3acdc2e642b4b3162d2c] does not match download [2a86aedfd5d06f761472e6f3590e840ad44ac1a5944a553af737fe7e0bd365d5]

**Comment by stephengillie** [comment]
Hi @rakleed,

This PR is using a vanity URL, and the hash has updated twice. Is the version still correct? Also, is there a non-vanity URL available?

**Comment by BrandonWanHuanSheng** [comment]
The tags suppose to put application number.

**Comment by rakleed** [comment]
> Hi @rakleed,
> 
> This PR is using a vanity URL, and the hash has updated twice. Is the version still correct? Also, is there a non-vanity URL available?

@stephengillie fixed version. I couldn't find non-vanity URL.

**Comment by stephengillie** [comment]
image

File: manifests/s/Stenzek/DuckStation/0.1.6232/Stenzek.DuckStation.installer.yaml [hunk 2]
```diff
@@ -1,16 +1,16 @@
 # Created using wingetcreate 1.2.5.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
 
-PackageIdentifier: stenzek.DuckStation
-PackageVersion: 20230128-gc6a5727
+PackageIdentifier: Stenzek.DuckStation
+PackageVersion: 0.1.6232
```
Comment (stephengillie) [inline manifests/s/Stenzek/DuckStation/0.1.6232/Stenzek.DuckStation.installer.yaml:5]:
```suggestion
PackageVersion: 0.1.6232
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 1/3/2024 8:24:25 PM
 Installer Verification Analysis Context Information: 
 Specified hash doesn't match.
 Actual hash:   8B02DDF97F4CAB05C37EDFF990DAE00CEF118DAF993F66117F0E2D667C5D7557
 Expected hash: 2a86aedfd5d06f761472e6f3590e840ad44ac1a5944a553af737fe7e0bd365d5

(Automated response - build 516.)

File: manifests/s/Stenzek/DuckStation/0.1.6232/Stenzek.DuckStation.installer.yaml [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
 - InstallerUrl: https://github.com/stenzek/duckstation/releases/download/latest/duckstation-windows-x64-release.zip
   Architecture: x64
   InstallerType: zip
   NestedInstallerType: portable
   NestedInstallerFiles:
   - RelativeFilePath: duckstation-qt-x64-ReleaseLTCG.exe
-    PortableCommandAlias: duckstation
-  InstallerSha256: BE682525515184C7CD829836014EA06569EA215930C4D796A25FD0B33CF06DE9
+    PortableCommandAlias: DuckStation
+  InstallerSha256: 2A86AEDFD5D06F761472E6F3590E840AD44AC1A5944A553AF737FE7E0BD365D5
```
Comment (stephengillie) [inline manifests/s/Stenzek/DuckStation/0.1.6232/Stenzek.DuckStation.installer.yaml:17]:
```suggestion
  InstallerSha256: 8B02DDF97F4CAB05C37EDFF990DAE00CEF118DAF993F66117F0E2D667C5D7557
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**